apps/web/src/components/templates/email-editor/EmailContentCard.tsx

```diff
@@ -80,24 +94,84 @@ export function EmailContentCard({
   return (
     <>
       {!isIntegrationActive ? <LackIntegrationError channelType="E-Mail" /> : null}
-      <Controller
-        name={`steps.${index}.template.subject` as any}
-        control={control}
-        render={({ field, fieldState }) => {
-          return (
-            <Input
-              {...field}
-              mb={40}
-              error={fieldState.error?.message}
-              label="Subject line"
-              disabled={readonly}
-              value={field.value}
-              placeholder="Type the email subject..."
-              data-test-id="emailSubject"
-            />
-          );
+      <div
+        style={{
+          fontWeight: 'bolder',
+          marginBottom: '10px',
         }}
-      />
+      >
+        Inbox View
+      </div>
+      <div
+        style={{
+          background: theme.colorScheme === 'dark' ? colors.B17 : colors.B98,
+          borderRadius: '7px',
+          marginBottom: '40px',
+          padding: '5px 10px',
+        }}
+      >
+        <Grid grow align="center">
+          <Grid.Col span={3}>
+            <div
+              style={{
+                padding: '15px',
+                borderRadius: '7px',
+                border: `1px solid ${theme.colorScheme === 'dark' ? colors.B30 : colors.B80}`,
+                margin: '5px 0px',
+              }}
+            >
+              {integration?.credentials?.from}
```

Comment:
What about the case of no integration? What should we show?

Reply:
Great question...